icu4c/source/common/util.cpp

```diff
@@ -65,6 +65,39 @@ UnicodeString& ICU_Utility::appendNumber(UnicodeString& result, int32_t n,
     return result;
 }
 
+void ICU_Utility::appendNumber(CharString &word, int32_t number, UErrorCode &status) {
```

Comment:
Put this logic in one place or the other.  Don't duplicate it

Reply:
As I mentioned in another comment, I made both of places for the discussion.

Removed!